.github/workflows/promote.yaml

```diff
@@ -0,0 +1,32 @@
+# reusable workflow triggered manually
+name: Promote charm to other tracks and channels
+
+on:
+  workflow_dispatch:
+    inputs:
+      destination-channel:
+        description: 'Destination Channel'
+        required: true
+      origin-channel:
+        description: 'Origin Channel'
+        required: true
+      charm-name:
+        description: 'Charm subdirectory name'
+        required: true
+
+jobs:
+  promote-charm:
+    name: Promote charm
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Release charm to channel
+        # TODO: use canonical/charming-actions/promote-charm?
```

Comment:
remove this comment, we've decided we don't to to use promote-charm as it's for multi-arch charms